docs/user-guide/incident-commander.md

```diff
@@ -6,61 +6,104 @@ description: What to expect as an incident commander.
 
 ## Reporting
 
-Within Dispatch Incident Commander's \(ICs\) are also participants and will receive all of the messaging that a participant does. When resolved as incident commander you are assigned that role by Dispatch and your identity is propagated via various messaging. Additionally for `High` severity events, the Incident Commander is automatically paged \(via PagerDuty\).
+Within Dispatch, Incident Commanders \(ICs\) are also participants, and will receive all of the participant messaging as well. When resolved as the Incident Commander, you are assigned that role by Dispatch, and your identity is propagated via various messaging. For `High` priority incidents, the Incident Commander is automatically paged \(via PagerDuty\).
```

Comment:
IC is only paged via the oncall service defined if the `Page Commander` checkbox is checked for the chosen incident priority when the incident is reported.
```suggestion
Within Dispatch, Incident Commanders \(ICs\) are also participants, and will receive all of the participant messaging as well. When resolved as the Incident Commander, you are assigned that role by Dispatch, and your identity is propagated via various messaging.
```